test/print_test_stats.py

```diff
@@ -46,11 +47,12 @@ def print_report(self):
         sorted_tests = sorted(self.test_cases, key=lambda x: x.time)
         test_count = len(sorted_tests)
         print(f"class {self.name}:")
-        print(f"    tests: {test_count} failed: {self.failed_count} skipped: {self.skipped_count} errored: {self.errored_count}")
+        print(
```

Comment:
can you revert this and all other format changes done automatically by the IDE. Seems like the report file
`test-reports` is also involved in the codecov CI job so there might've been some formatting issues here and there getting triggered.

Reply:
Thanks for pointing out! Changed it back.